### DISCUSSION TITLE
Does TurbSim support OpenMP?

### DISCUSSION THREAD

**Discussion by Zhiming-Cai** [OpenFAST/openfast#1411] [Q&A]
When I was generating wind with TurbSim, I noticed that the TurbSim process was only using one thread for computation, but I had OpenMP support turned on in <CMakeLists.txt> while compiling. Could you tell me if TurbSim supports OpenMP please？

Settings in <CMakeLists.txt>👇
<img width="690" alt="屏幕截图 2023-01-17 124152" src="https://user-images.githubusercontent.com/89171153/212811832-c5432580-4910-49c9-9c82-73713bc461d8.png">

<top>  command👇
<img width="354" alt="屏幕截图 2023-01-17 124232" src="https://user-images.githubusercontent.com/89171153/212812023-34a099a3-5255-45da-8311-1d7e4941eaab.png">

@jjonkman @bjonkman @NRELAdmin

**Comment by bjonkman** [comment]
The TurbSim source code does not yet have any OpenMP directives, so building it with OpenMP support will not result in increased performance.
